section11/numberformat_proposed.html

```diff
@@ -621,6 +630,30 @@ <h1>GetNumberFormatPattern ( _numberFormat_, _x_, _exponent_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-getnotationsubpattern" aoid="GetNotationSubPattern">
+      <h1>GetNotationSubPattern ( _numberFormat_, _exponent_ )</h1>
+      <p>
+        The abstract operation GetNotationSubPattern considers the resolved notation and exponent, and returns a String value for the notation sub pattern as described in <emu-xref href="#sec-intl.numberformat-internal-slots"></emu-xref>. 
```

Comment:
nit
```suggestion
        The abstract operation GetNotationSubPattern considers the resolved notation _numberFormat_ and _exponent_, and returns a String value for the notation sub pattern as described in <emu-xref href="#sec-intl.numberformat-internal-slots"></emu-xref>. 
```

Reply:
`_numberFormat_` is the intl object, not the notation itself.  I'll stylize `_exponent_` though.  If we want to stylize notation, we should do something like `_numberFormat_.[[Notation]]`, but I find that less readable.